.github/workflows/nuget.yml

```diff
@@ -2,19 +2,21 @@ name: Windows Nuget Build
 
 on:
   push:
+    branches: [ master ]
     paths-ignore:
       - 'CHANGES*'
       - 'Doc/**'
       - 'appveyor.yml'
   pull_request:
-    branches: master
+    branches: [ master ]
     paths-ignore:
       - 'CHANGES*'
       - 'Doc/**'
       - 'appveyor.yml'
 
-permissions:
-  contents: read  #  to fetch code (actions/checkout)
```

Comment:
This was deliberately added by 62e27bb61680482ec9a1a0a27f7bb29946c9472e.

It seems this is just setting the default, but I assume the idea was it's better to explicitly set it to the value we want rather than rely on the default.  I suggest you should discuss with @sashashura as this part is effectively reverting their changes.

Reply:
Oh, I've had a read and I've got the permission mixed up. I thought it was additive, not exclusive. I'll revert and add minimal comments for it